src/commands/remote.ts

```diff
@@ -1,5 +1,33 @@
-import { NamedMessage } from '../message'
-import { AbstractCommandNoResponse } from './abstractCommand'
+import { SynchronousCode } from '../codes'
+import { ResponseMessage, NamedMessage } from '../message'
+import { parseBool } from '../util'
+import { AbstractCommand, AbstractCommandNoResponse } from './abstractCommand'
+
+export interface RemoteInfoCommandResponse {
+	enabled?: boolean
+	override?: boolean
+}
+
+//TODO A response is optional, may be a 200. How do we handle this?
```

Comment:
This todo seems unnecessary, since get and set are separated commands now.

Reply:
Indeed it is, and has been removed.